Makefile

```diff
@@ -12,11 +15,26 @@ dependencies: ## Install dependencies
 .PHONY: build
 build: ## Builds the project
 	@echo "Run middleman..."
-	bundle exec middleman build
+	bundle exec middleman build --verbose
 	@echo "Copy additional files..."
 	cp googlec7239f490e1990a5.html build
 
 .PHONY: test
-test: ## Runs the tests
-	bundle exec middleman build --verbose
-	linkchecker ./build/
+test: test-filesystem test-local-http ## Runs the tests
```

Comment:
Does `test-filesystem` cover anything that `test-local-http` doesn't?  If not, is there any value in running both here?

Reply:
No. But, as per the comment introducing it in #178, test-filesystem is a fail-fast test and local-http takes somewhat longer. We want CI to run the more rigorous tests (and I didn't want to modify CI's pipeline as this is a bit of a drive-by), so giving devs the option of one test, the other, or both, seemed sufficient.

Reply:
(BTW the reason you're seeing this target introduced *here* is because I preemptively rebased this branch onto #178, where it's **actually** introduced)